opengate/actors/doseactors.py

```diff
@@ -203,43 +233,133 @@ def EndSimulationAction(self):
         # in the coordinate system of the attached volume
         # FIXME no direction for the moment ?
         self.py_edep_image.SetOrigin(self.output_origin)
+
+        # dose in gray
+        if self.user_info.dose:
```

Comment:
This is on the one hand legacy, on the other hand is useful to add a suffix to indicate the type of output. Another way could be to write the type AND unit into the mhd header? 

There is another advantage of not touching the filename/path of the output, so we do not need to ask for the DoseActor output object to get the filename, which is actually annoying. 
So from my side I am fine with any

Reply:
I like the idea of putting the information into the header file. That is what a header file is there for in the end and the information would not get lost in case the user later changes the filename. I'll check how to best put this info into the header. 


And yes, it's annoyingly hacky to fiddle with the filenames.

Reply:
Actually we could easily support both, no? If we require the renaming of the path to be done within one single function/method we could control the behavior with a global flag. By default the renaming (= adding suffix for specific type of simulation) could be turned off, some users may turn it on for convenience? (although in python this task is so trivial, that this feature may be an overkill )

A useful feature in the future could be to have auto fname generation e.g. {actor_name}-{actor_subtype}-{volume_attached_to}-{filters_applied}-{...}

Reply:
@andiresch and @MartiDvrk 
todo; move it to path object handler: simulation.get_output_path